### ISSUE TITLE
Connection to 'xxx' was not upgraded to websocket

### ISSUE THREAD

**Issue by AndroidZern** [cloudwebrtc/dart-sip-ua#70]
我运行了flutter，运行在ios上面注册不上。

**Comment by thelittlebug** [comment]
i have exactly the same issue with my freeswitch

my html/js based webrtc client is able to connect, dart-sip-ua not.

**Comment by thelittlebug** [comment]
i tried to connect to my freeswitch wss url with this browser extension: https://chrome.google.com/webstore/detail/websocket-test-client/fgponpodhbmadfljofbimhhlengambbn/related

and it worked

maybe a hint:
Level.debug websocket_interface.dart:94 ::: Closed [500, WebSocketException: Connection to 'https://xxx.xxx.xxx:39443/#' was not upgraded to websocket]!

but where does the # comes from?

**Comment by thelittlebug** [comment]
in my case im pretty sure its related to this issue: https://github.com/dart-lang/sdk/issues/25120

tl;dr:
dart is automatically converting http headers to lowercase
legacy servers are not ok with that

**Comment by thelittlebug** [comment]
today i upgraded my freeswitch installation to 1.10
now it works as expected because 1.10 handles the headers case insensitive.

**Comment by ghenry** [comment]
Thanks for the follow up @thelittlebug
